### PR TITLE
Feat/convert h and g files ts

### PR DESCRIPTION
**Pull request by vdusart**
Moving files starting with letter "G" and "H" to TS

# Related Issue

Conversion to TS according to #6392

### CODE REVIEW

File: src/components/GitStars.tsx [hunk 1]
```diff
@@ -46,8 +46,18 @@ const Text = styled.div`
...
 `
+export interface GitHubRepo {
+  stargazerCount: number
+  url: string
+}
+
+export interface IProps {
+  gitHubRepo: GitHubRepo
+  className?: string
+  hideStars: string
```
Comment (pettinarip) [inline src/components/GitStars.tsx:57]:
```suggestion
  hideStars: boolean
```

**Review by pettinarip — CHANGES_REQUESTED** [review]
Nice! some minor comments, thanks again @vdusart 💪🏼

File: src/components/FeedbackCard.tsx [hunk 2]
```diff
@@ -39,7 +39,12 @@ const ButtonContainer = styled.div`
   }
 `
 
-const FeedbackCard = ({ prompt, className }) => {
+export interface IProps {
+  prompt?: string
+  className?: string
+}
+
+const FeedbackCard: React.FC<IProps> = ({ prompt, className }) => {
```
[the remark this answers is not in the document]
Comment (vdusart) [reply src/components/FeedbackCard.tsx:47]: Yeah you're right I don't know what happened I pushed this commit twice sorry :smile:

**Review by pettinarip — APPROVED** [review]
gg lfg! @vdusart Thanks!

**Pull request merged**